sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,906 @@
+				axes.plot_date(dates, data['col' + str(7+i)].ffill(), '-', label= 'Sector '+str(i), color=colors[i/2-2], lw=2)
+
+		elif header[2][:22] == 'Column 7: LET Livetime':
+			#Non sectored standard data
+			num_energy_bins = len(header) - 4
+			for i in range(num_energy_bins):
+				data = data.replace(-1.000000e+31,float('nan'))
+				# print 'col' + str(8+i), data['col' + str(8+i)]
+				axes.plot_date(dates, data['col' + str(8+i)].ffill(), '-', 
+						label= header[3+i][header[3+i].index(':')+1:header[3+i].index('/')+2], color=colors[i], lw=2)
+		else:
+			#Non sectored non standard data
+			num_energy_bins = len(header) - 4
+			for i in range(num_energy_bins):
+				data = data.replace(-1.000000e+31,float('nan'))
+				# print 'col' + str(8+i), data['col' + str(8+i)]
+				axes.plot_date(dates, data['col' + str(7+i)].ffill(), '-', 
+						label= header[2+i][header[2+i].index(':')+1:header[2+i].index('/')+4], color=colors[i], lw=2)
+
+
+		
+		axes.set_title(title)
+		axes.set_ylabel('1/(cm^2 s sr MeV/nuc)')
+		axes.set_xlabel('UTC TimeZone')
+
+		axes.yaxis.grid(True, 'major')
+		axes.xaxis.grid(False, 'major')
+
+		axes.legend(loc='center left', bbox_to_anchor=(1, 0.5))
+
+		figure.autofmt_xdate()
+		plt.show()
+
+		return figure
+
+	@staticmethod
+	def _parse_txt(filepath):
+		"""
+		Parses a STEREO LET file from
+		http://www.srl.caltech.edu/STEREO/Public/LET_public.html
+
+		""" 
+		type_of_data = ''
+		header = []
+		data_read = {'data_type_read': False, 'header_line_reached': False }
+
+		#To Read in header and get index of line where data starts
+		fp = open(filepath)
+		for i,line in enumerate(fp):
+			#To Find the type of data, 27 day, 1 day, 1 hour, 10 min or 1 min
+			if data_read['data_type_read'] == False:
+				if i == 2 and line[:3] == 'Bin':
+					type_of_data = '27day'
+					data_read['data_type_read'] = True
+				elif i in [5,6] and line[:4] == 'Time': 
+					type_of_data = line[17:].strip('\n')
+					data_read['data_type_read'] = True
+				else:
+					continue
+
+			# To figure out and customise header in case of 27 day type of data
+			if type_of_data == '27day':
+				if line[:3] == 'Bin':
+					header = header + [line]
+					continue
+				elif line[:7] == 'Columns':
+					data_start = i + 4
+					data_points = int(line[20:line.index('f')-1])
+				
+					for k in range(data_points):
+						header = header + ['Uncertainty for ' + header[k]]
+						header[k] = 'Flux for ' + header[k]
+					
+					header = ['Datetime'] + header
+					break
+			   
+			# To figure out and customise header in case of any other type of data since their files are formatted 
+			# similarly but different from 27 day type of data
+			else:
+				if data_read['header_line_reached'] == False and line[:6] != 'Column':
+					continue
+				elif line[:6] == 'Column':
+					header = header + [line]
+					data_read['header_line_reached'] = True
+					continue
+				elif line == 'BEGIN DATA\n':
+					data_start = i+1
+					break      
```

Comment:
I'm almost sure this `if` block can be reduced to just a `if` and `elif` and remove the first `continue`